### DISCUSSION TITLE
[Question] Prusa MK3 WiFi Use Case with OctoPrint

### DISCUSSION THREAD

**Discussion by jwillmer** [luc-github/ESP3D#772] [Questions]
I have a Prusa MK3 and like to enable a wifi connection to it in order to monitor the print telemetry over the network. I used a raspberry pi in the past but the hard shutdown of the pi when killing the power on the printer destroyed my operating system. Additionally I had to wait for it to boot up. 

My question is: Is this project feasible to connect my printer to my wifi, host OctoPrint on a dedicated server and be able to connect to my printer (monitor/start/stop) it using OctoPrint?

**Comment by luc-github** [comment]
This Firmware do not host  host octoprint - it is specific ESP FW which can control/monitor  printer  thrue webUI https://github.com/luc-github/ESP3D-WEBUI or wifi using raw tcp (printerface / repetier host)

**Reply by jwillmer** [reply]
Yes, of course. OctoPrint would connect via Lan. Isn't this project suppose to work like this?

**Reply by luc-github** [reply]
the webui is hosted in esp board 
the applications communicating by telnet are on PC

**Reply by jwillmer** [reply]
Ah, sorry. Then I misunderstood the project goal. I'm looking for some bridge that can connect my printer with my WIFI so that I can host the 3D print controller somewhere else.

**Reply by luc-github** [reply]
yes this is what I wrote above 
you can have repetier host or pronterface  on your PC to control the printer communicating using raw tcp 
But even it is possible I won't suggest stream GCODE using wifi due to network latency, better limit to monitor and control printer

**Reply by jwillmer** [reply]
Found this project an hour ago. Might solve my use case :) https://github.com/probonopd/WirelessPrinting

**Reply by luc-github** [reply]
great, have happy printing :)
